mirrord/agent/src/sniffer.rs

```diff
@@ -268,6 +269,8 @@ where
                 command: SnifferCommandInner::NewClient(sender),
             } => {
                 self.handle_new_client(client_id, sender);
+
+                MIRROR_CONNECTION_SUBSCRIPTION.inc();
```

Comment:
I don't see corresponding `.dec()` anywhere

Reply:
Added to `handle_client_closed`.